os/net/lwip/src/include/lwip/opt.h

```diff
@@ -1771,7 +1771,7 @@
  * SO_SNDTIMEO processing.
  */
 #ifndef LWIP_SO_SNDTIMEO
-#define LWIP_SO_SNDTIMEO                0
```

Comment:
Could you explain why you changed this to 1? If possible, please add description in commit msg or comment.

Reply:
explained in commit msg

Reply:
this is wrong approach. if you want to enable LWIP_SO_SNDTIMEO then you have to configure menuconfig 'NET_SO_SNDTIMEO' value.
If you set NET_SO_SNDTIMEO then LWIP_SO_SNDTIMEO will be enabled. Please check lwipopts.h that I explained before.
~~~
#ifdef CONFIG_NET_SO_SNDTIMEO
#define LWIP_SO_SNDTIMEO	CONFIG_NET_SO_SNDTIMEO
#endif
~~~